pkg/volume/gce_pd/gce_util.go

```diff
@@ -258,6 +285,37 @@ func detachDiskAndVerify(c *gcePersistentDiskCleaner) {
 	glog.Errorf("Failed to detach GCE PD %q. One or more mount paths was not removed.", c.pdName)
 }
 
+// Returns true if the operation with the specified operationId completed.
+func verifyOperationCompletion(gce cloudprovider.Interface, operationId string) (bool, error) {
```

Comment:
Why not pass around `*gce_cloud.GCECloud` and just convert and assert once up-stack?